tests/integration/authentication.test.js

```diff
@@ -23,7 +23,7 @@ describe('POST /users/authenticate', () => {
     const response = await app.inject({
       method: 'POST',
       url: '/users/authenticate',
-      payload: { email: 'kitty@alenvi.io', password: '123456!eR', origin: 'webapp' },
+      payload: { email: 'carolyn@alenvi.io', password: '123456!eR', origin: 'webapp' },
```

Comment:
pourquoi ce changement?

Reply:
ça me permet de supprimer le mot de passer pour le l'utilisateur kitty@alenvi.io dans les seeds, ça permet de gagner un peu en perf

Reply:
dans ce cas, il faut peut etre que tu changes aussi l'email dans le test `it('should not authenticate a user if wrong password', async () => {` pour utiliser un compte qui a bien un mot de passe defini

Reply:
bien vu ! j'ai modifié un autre test d'authentification du coup